### PR TITLE
Done BinarySearch tree in java

### LINKED ISSUE #77: Binary Search Tree implementation
# Binary Search Tree

Implement a Binary Search Tree with all basic methods like traversals, update, deletion, addition, and finding of data.

# Assignees:

- C: @Harshalszz
- C++: @keerti2001
- Python:
- Java

### PR DESCRIPTION
**Pull request by vnishchay**
Please check. Completed Binary Search tree implementation in java

fix #77

### CODE REVIEW

File: Java/binarySearchTree.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
+class BST_class {
```
Comment (kaustubhgupta) [inline Java/binarySearchTree.java:4]: Update the language folder readme

File: Java/binarySearchTree.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            return search_Recursive(root.left, key); 
+        // val is less than root's key 
+        return search_Recursive(root.right, key); 
+    } 
+}
+class Main{
+    public static void main(String[] args)  { 
+       //create a BST object
+        BST_class bst = new BST_class(); 
+        /* BST tree example
```
Comment (kaustubhgupta) [inline Java/binarySearchTree.java]: Remove this whole doc string

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
@codernishchay this look at another review I gave for this program code (Update the readme of language folder)

**Comment by vnishchay** [comment]
Ok. I will do it by today

**Pull request merged**